src/styles/general.css

```diff
@@ -1,9 +1,28 @@
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@400;500;700&family=Roboto+Slab:wght@300;700&display=swap');
+
+body {
+  background-color: #f5f6fa;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+#root {
+  width: 80%;
+  min-height: 80% !important;
+  margin-top: 4rem;
```

Comment:
- [ ] Kindly it will be better 😃 if you remove the ```!important``` from the style here as this is from basic CSS bad practices are not used, so The ```!important``` rule was conscripted in the mid-to-late 90’s to help web designers and developers override the rules of their style sheet. The ```!important``` rule💯 prevails over all the previous styles and says, “I’m more important, forget about all of that other CSS and use me!“. Using this out of laziness will give you a headache later on as your style sheet will be harder to maintain.🚀👌You can read [Here](https://speckyboy.com/good-bad-css-practices) about best practices 😊💯